release.sh

```diff
@@ -1,13 +1,21 @@
 #!/bin/bash
 
+UPSTREAM_URL='git@github.com:TNG/momo-scheduler.git'
+# default: not dry-run
+DRY_RUN=0
+UPSTREAM_BRANCH=main
+
 set -e
```

Comment:
suggestion: set -eux
u treats unset variables as error
x prints every command that is executed